blog/.gitignore

```diff
@@ -1,2 +1,3 @@
 _site
 .sass-cache
+.jekyll-metadata
```

Comment:
Added `/public/blog` to the Rails app's .gitignore file one level up in the file tree - seems like the .gitignore inside the Jekyll app at `/blog` only monitors files inside of that directory.